content/en/docs/concepts/overview/components.md

```diff
@@ -74,58 +108,53 @@ The following controllers can have cloud provider dependencies:
   * Route controller: For setting up routes in the underlying cloud infrastructure
   * Service controller: For creating, updating and deleting cloud provider load balancers
 
-## Node Components
+## Add-ons {#addons}
 
-Node components run on every node, maintaining running pods and providing the Kubernetes runtime environment.
-
-### kubelet
-
-{{< glossary_definition term_id="kubelet" length="all" >}}
-
-### kube-proxy
-
-{{< glossary_definition term_id="kube-proxy" length="all" >}}
-
-### Container Runtime
-
-{{< glossary_definition term_id="container-runtime" length="all" >}}
-
-## Addons
-
-Addons use Kubernetes resources ({{< glossary_tooltip term_id="daemonset" >}},
+Add-ons use Kubernetes resources ({{< glossary_tooltip term_id="daemonset" >}},
 {{< glossary_tooltip term_id="deployment" >}}, etc)
 to implement cluster features. Because these are providing cluster-level features, namespaced resources
-for addons belong within the `kube-system` namespace.
+for addons often belong within the `kube-system`
+{{< glossary_tooltip term_id="namespace" text="namespace">}}.
 
-Selected addons are described below; for an extended list of available addons, please
-see [Addons](/docs/concepts/cluster-administration/addons/).
+Some important examples of add-ons are:
 
 ### DNS
 
-While the other addons are not strictly required, all Kubernetes clusters should have [cluster DNS](/docs/concepts/services-networking/dns-pod-service/), as many examples rely on it.
+While the other add-ons are not strictly required, all Kubernetes clusters should have [cluster DNS](/docs/concepts/services-networking/dns-pod-service/), as many elements of Kubernetes rely on it.
 
-Cluster DNS is a DNS server, in addition to the other DNS server(s) in your environment, which serves DNS records for Kubernetes services.
+_Cluster DNS_ provides DNS specifically for your Kubernetes cluster, and lets Pods in the
+cluster look up DNS records for Kubernetes services.
 
-Containers started by Kubernetes automatically include this DNS server in their DNS searches.
+When Kubernetes runs a container, the kubelet provides configuration so that the application
+in the container automatically includes the cluster DNS service in its DNS searches.
 
 ### Web UI (Dashboard)
 
 [Dashboard](/docs/tasks/access-application-cluster/web-ui-dashboard/) is a general purpose, web-based UI for Kubernetes clusters. It allows users to manage and troubleshoot applications running in the cluster, as well as the cluster itself.
 
-### Container Resource Monitoring
+### Resource monitoring
 
-[Container Resource Monitoring](/docs/tasks/debug-application-cluster/resource-usage-monitoring/) records generic time-series metrics
-about containers in a central database, and provides a UI for browsing that data.
+[Resource monitoring tools](/docs/tasks/debug-application-cluster/resource-usage-monitoring/) let you
+records generic time-series metrics about pods and their containers. You can link these up
```

Comment:
s/record/records/
Reword: `You can link these up`

Reply:
`s/records/record/`, right?

Reply:
Yep, fast typing.